flask/test/endpoint_tests/conftest.py

```diff
@@ -1,12 +1,14 @@
 """Functionality for common test setups.
-See https://docs.pytest.org/en/stable/fixture.html#conftest-py-sharing-fixture-functions
+See https://docs import
+test.org/en/stable/fixture.html#conftest-py-sharing-fixture-functions
```

Comment:
Is this on purpose?

Reply:
the precommit config isnt working and complains about line width at the moment I was trying to get a commit in and shortening some of the lines. Not needed I'll revert